class Error(Exception):
    pass


class OpCallError(Error):
    pass
